BCnEnc.Net/Encoder/RawEncoders.cs

```diff
@@ -139,4 +139,36 @@ public uint GetGlTypeSize()
 
 		public DxgiFormat GetDxgiFormat() => DxgiFormat.DxgiFormatR8G8B8A8Unorm;
 	}
+
+	internal class RawBgraEncoder : IRawEncoder
+	{
+		public byte[] Encode(ReadOnlySpan<Rgba32> pixels)
+		{
+			var output = new byte[pixels.Length * 4];
+			for (var i = 0; i < pixels.Length; i++)
+			{
+				output[i * 4] = pixels[i].B;
+				output[i * 4 + 1] = pixels[i].G;
+				output[i * 4 + 2] = pixels[i].R;
+				output[i * 4 + 3] = pixels[i].A;
+			}
+			return output;
+		}
+
+		public GlInternalFormat GetInternalFormat()
```

Comment:
Please use the same style as in the other encoders by using brackets and ``return``. This makes it look more consistent.